osu.Game.Tests/Resources/skin-empty.ini

```diff
@@ -1,2 +0,0 @@
-[General]
-Name: test skin
```

Comment:
what's the reason for removing this?

Reply:
Doesn't seem to be used, although i've realized it has a usage with something i'm working on, will return.